CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 - Added code logic to send `VIRTIO_VSOCK_EVENT_TRANSPORT_RESET` on snapshot
   creation, when the Vsock device is active. The event will close active
   connections on the guest.
+- Added `GET` request on `/export-vm-config` that provides full microVM
+  configuration as a JSON HTTP response.
```

Comment:
Can the config be used as JSON to start FC ?

Reply:
Yes, it's using the same JSON serialized struct as the `--config-file` param uses 👍

Reply:
Then, we also need a test that ensure this -> new VM -> dump config -> new VM using dumped config.

Reply:
I wouldn't say we _need_ that test, but sure, will add.

Reply:
> Then, we also need a test that ensure this -> new VM -> dump config -> new VM using dumped config.

Did you just uncover a new way of starting VMs? 😁

Reply:
I think so ... an easy way to convert from "curl" based configuration to one shot json based startup 😁

Reply:
done! 👍